pisound improvements:

gtrainavicius opened the pull request:
* Added a writable sysfs object to enable scripts / user space software to blink MIDI activity LEDs for variable duration.
* Improved hw_param constraints setting.
* Added compatibility with S16_LE sample format.
* Exposed some simple volume controls, so the card appears in volumealsa widget.

gtrainavicius committed:
pisound improvements:

* Added a writable sysfs object to enable scripts / user space software
to blink MIDI activity LEDs for variable duration.
* Improved hw_param constraints setting.
* Added compatibility with S16_LE sample format.
* Exposed some simple placeholder volume controls, so the card appears
in volumealsa widget.

Signed-off-by: Giedrius Trainavicius <giedrius@blokas.io>

pelwell commented:
Do the controls do anything yet? They look like read-only placeholders.

If you add a Signed-off-by: line to your commit message and split the long lines (Linux likes the message lines to be 75 characters or less) - or give me permission to do that for you - then I'll merge it (unless @popcornmix objects).

gtrainavicius commented:
Thank you for your feedback! Restructured the commit message lines to fit in ~74 chars, added mention that the volume controls are placeholders (the volumealsa widget will just show full volume if the card is selected), it was necessary to include them in order for the card to be listed by volumealsa widget in the first place. Added Signed-off-by.

pelwell commented:
Looks good to me.
